assets/js/local.timeslot.js

```diff
@@ -136,8 +136,7 @@
                 time: this.$el.find(this.timeSelector).val()
             }
         }).done(function () {
-            self.fillDates()
-            self.togglePicker()
+            location.refresh();
```

Comment:
shouldn't this be `location.reload(true)`?

Reply:
Yep sorry, wasnt thinking

Reply:
hehe, happens.